lib/chat_api_web/controllers/registration_controller.ex

```diff
@@ -80,6 +80,34 @@ defmodule ChatApiWeb.RegistrationController do
     conn
   end
 
+  defp send_registration_event(conn) do
+    with %{email: email, account_id: account_id} <- conn.assigns.current_user do
+      %{email: email}
+
+      now = :os.system_time(:seconds)
+
+      param = %{
+        email: email,
+        created_at: now,
+      }
+
+
+      Customerio.identify(account_id, param)
```

Comment:
just curious, why account ID and not user ID here?

Reply:
Good catch actually should be user_id